fe/fe-core/src/main/java/org/apache/doris/spi/Split.java

```diff
@@ -15,17 +15,15 @@
 // specific language governing permissions and limitations
 // under the License.
 
-package org.apache.doris.planner;
+package org.apache.doris.spi;
 
-import lombok.Data;
+/**
+ * Split interface. e.g. Tablet for Olap Table.
+ */
+public interface Split {
 
-@Data
-public abstract class Split {
-    protected String[] hosts;
+    String[] getHosts();
 
-    public Split() {}
+    Object getInfo();
```

Comment:
What is this for?

Reply:
There may be significant variance between different splits, this reserved interface may may useful in the future. But for now, it is useless.